app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1623,6 +1628,29 @@ public void clearChatBox() {
     }
 
     @Override
+    public void setLayoutBehaviour(LinearLayoutManager linearLayoutManager, RecyclerMessageAdapter adapter) {
+        CoordinatorLayout.LayoutParams layoutParams = (CoordinatorLayout.LayoutParams)appBarLayout.getLayoutParams();
+        layoutParams.setBehavior(new RemoveViewsOnScroll(linearLayoutManager, adapter));
+        appBarLayout.requestLayout();
+
+        layoutParams = (CoordinatorLayout.LayoutParams) fab.getLayoutParams();
+        layoutParams.setBehavior(new RemoveViewsOnScroll(linearLayoutManager, adapter));
+        fab.setLayoutParams(layoutParams);
+
+        topSnackBar.setMessagesLayoutManager(linearLayoutManager);
+        topSnackBar.setMessagesAdapter(adapter);
+    }
+
+    private int statusBarHeight(){
```

Comment:
Its returning height of the status bar. Now I have updated and removed this method.